### ISSUE TITLE
Adding details for migraine 

### ISSUE THREAD

**Issue by Gaminger** [rogue-elephant/migraine-tracker#8]
A user should be able to to add the following to an event;
 Migraine severity 1-10
Have you drank enough today? Yes/No
Are you stressed/angry? Yes /No
What foods have you had? Multiple choice of: Lactose, caffeine, wheat, sugar etc (because free txt box is difficult to do analytics on later)

**Comment by rogue-elephant** [comment]
Yes, already have these in the app - have you drunk enough I think would have to become something like "how many cups of water have you drunk"

I am thinking to allow free text boxes and then use a machine learning package to pull out the key terms see #4  for details as well.
